client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -749,15 +749,21 @@ export default {
         },
         sort: this.selectedSort,
         include: [
-          'segments',
           'assignee',
           'sourceAttachment',
           'sourceAttachment.file'
         ].join(),
         fields: {
           Statement: statementFields.join(),
           SourceStatementAttachment: [
+            'attachmentType',
             'file'
+          ].join(),
+          File: [
+            'hash',
```

Comment:
Do we really need more than the `hash`?

I also noticed that when I try to open an "Original-PDF", I sometimes get an error ("kein Inhalt gefunden") - but I don't know if it's because of these changes

Reply:
you receive the pdf error in case dplan cannot find the file on your machine. I do not think that this is a regression here. you are correct about the other properties, hash is sufficient [57b6221](https://github.com/demos-europe/demosplan-core/pull/5266/commits/57b6221d31d2d923f79045af7613434b4bf452c8)